src/stream.h

```diff
@@ -50,4 +50,14 @@ GIT_INLINE(void) git_stream_free(git_stream *st)
 	st->free(st);
 }
 
+/* Macros for locking around OpenSSL and libssh2; it assumes you have a static 'should_lock' variable */
+#define LOCK do {						   \
```

Comment:
Because it's not meant to be a generally-useful macro, but one for this exact use.